pkg/scheduler/eventhandlers.go

```diff
@@ -139,21 +139,21 @@ func (sched *Scheduler) deleteNodeFromCache(obj interface{}) {
 		var ok bool
 		node, ok = t.Obj.(*v1.Node)
 		if !ok {
-			klog.Errorf("cannot convert to *v1.Node: %v", t.Obj)
+			klog.ErrorS(nil, "cannot convert to *v1.Node", "obj", t.Obj)
 			return
 		}
 	default:
-		klog.Errorf("cannot convert to *v1.Node: %v", t)
+		klog.ErrorS(nil, "cannot convert to *v1.Node", "type", t)
 		return
 	}
-	klog.V(3).Infof("delete event for node %q", node.Name)
+	klog.V(3).InfoS("delete event for node %q", "node", klog.KObj(node))
```

Comment:
```suggestion
	klog.V(3).InfoS("delete event for node", "node", klog.KObj(node))
```

Reply:
Sorry, I left out some details. Thank you very much for the careful review. Done.